packages/api/cms-api/src/common/filter/mikro-orm.ts

```diff
@@ -274,10 +274,15 @@ export function searchToMikroOrmQuery(search: string, fieldsOrMetadata: string[]
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             const or: any = {};
             let nestedFilter = or;
-            for (const fieldPart of field.split(".")) {
+            const fieldParts = field.split(".");
```

Comment:
I tried to find a better name for fieldParts, e.g., columnPath, but I wasn't happy with any of them.